docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -114,3 +121,77 @@ def merge_predictions_and_gt(predictions, old_ground_truth, field_config):
 
     return updated_predictions
 
+
+def overlap(last_lines, first_lines):
+    for first_line in first_lines:
+        for last_line in last_lines:
+            if first_line['label'] == last_line['label'] and first_line['value'] != last_line['value']:
+                return False
+    return True
+
+
+def merge_lines(last_line, first_line):
+    return last_line + first_line
+
+
+def merge_lines_from_different_pages(predictions, field_config):
```

Comment:
I suggest we do a proper clean up of the code in another PR, where all of this can be fixed. I have made an issue: https://github.com/LucidtechAI/cradl-docs/issues/190